packages/commonwealth/server/migrations/20240425141332-add-unique-index-to-community-contracts.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface) => {
+    await queryInterface.sequelize.transaction(async (t) => {
+      await queryInterface.sequelize.query(
+        `
+ ALTER TABLE ONLY public."CommunityContracts"
+ ADD CONSTRAINT "CommunityContracts_community_id_contract_id_key" UNIQUE(community_id, contract_id)
```

Comment:
This creates an index on `(community_id, contract_id)` so we can remove the index on `community_id` since it becomes redundant.